### DISCUSSION TITLE
VirtualBox isn't found even though it is installed on my system.

### DISCUSSION THREAD

**Discussion by leothefleo49** [Brikwerk/nxbt#150] [Q&A]
Hey guys! I was trying to download this and use the software, but I'm not super  experienced with this. I was trying to put in the code to open the vagrant_setup.py but I couldn't figure that out so I just opened the vagrant_setup.py with visual studio code and ran it, but it didn't work. It says I don't have VirtualBox installed even though I do. I uninstalled it and reinstalled VirtualBox and the add on multiple times but nothing. Maybe I downloaded it to the wrong directory? But It also didn't seem to give me an option of where to download when I was downloading VirtualBox, but maybe I missed it.  The code says Vagrant is downloaded and working but idk how to fix the VirtualBox error. I'm on Windows 11 btw. So yeah, any help would be very very appreciated! Thanks! 
Here is what returns when I run the code through visual studio code:  
Checking for the required utilities...
Vagrant [OK]
VirtualBox [ERROR]
 -> VirtualBox wasn't found on your system
    VBoxManage (part of the VirtualBox CLI) wasn't found
    on your system path. Please ensure that VirtualBox is
    installed and VBoxManage is on your system path.
PS C:\Users\User>

**Comment by 2021Dumpsterfiredoescoding** [comment]
I have this same problem, I tried adding it to the system path (https://youtu.be/miC9HtiMxuM?si=3-vXoorpM8a3gGxT) but it still isn't working, if you got it to work please tell me how
